### PR TITLE
View playground info

### LINKED ISSUE #13: View-Playground-Info
A player should be able to view a playground info.
Request to add view a playground info feature option in the main menu of PlayerUI

### PR DESCRIPTION
**Pull request by Muhammad1995-djiddo**
Added a function to view a playground info in PlayerUI class
Added a function that returns a playground object if found or null otherwise in GoFo class

### CODE REVIEW

File: src/GOFO/UI/PlayerUI.java [hunk 1]
```diff
@@ -19,19 +19,59 @@ public PlayerUI(String id, Player player) {
...
+        System.out.print("Playground ID: ");
+        playgroundID = input.nextLine();
+
+        playground = GoFo.getPlaygroundInfo(playgroundID);
+
+        if (playground == null) {
+            System.out.println("\n==>No playground found with ID: " + playgroundID);
+            System.out.print("Would you like to try again?(Y/N): ");
+            String tryAgain = input.nextLine();
+            while (!tryAgain.equals("Y") && !tryAgain.equals("N")) {
```
Comment (ngchrbn) [inline src/GOFO/UI/PlayerUI.java]: You should accept lowercase character also and change it yourself

File: src/GOFO/UI/PlayerUI.java [hunk 2]
```diff
@@ -19,19 +19,59 @@ public PlayerUI(String id, Player player) {
...
+                System.out.println("Invalid input!");
+                System.out.print("Enter Y or N: ");
+                tryAgain = input.nextLine();
+            }
+            if (tryAgain.equals("N")) {
```
Comment (ngchrbn) [inline src/GOFO/UI/PlayerUI.java]: Add tryAgain.toUpperCase.equals("N")

**Review by ngchrbn — CHANGES_REQUESTED** [review]
Resolve the commented codes

File: src/GOFO/UI/PlayerUI.java (hunk 1 above)
Comment (Muhammad1995-djiddo) [reply src/GOFO/UI/PlayerUI.java]: I changed equals() to equalsIgnoreCase()

File: src/GOFO/UI/PlayerUI.java (hunk 2 above)
Comment (Muhammad1995-djiddo) [reply src/GOFO/UI/PlayerUI.java]: I changed equals() to equalsIgnoreCase()

**Review by ngchrbn — APPROVED** [review]
👍🏼

**Pull request merged**